### ISSUE TITLE
Build instructions?

### ISSUE THREAD

**Issue by RPMYT** [Kurausukun/mother3#63]
Title.
Seems to be missing `agbcp`, which `agbcc` does *not* include.

**Comment by Kurausukun** [comment]
Yes, you need to clone it separately; build instructions are very WIP right now because the repo isn't really usable for anything yet.

**Comment by blazingcody0707** [comment]
How to build this on arm64 linux termux proot debian trixie

**Comment by blazingcody0707** [comment]
<img width="1080" height="2340" alt="Image" src="https://github.com/user-attachments/assets/b38b7adc-8207-40b5-9ce1-1373d3023e3f" />

**Comment by blazingcody0707** [comment]
I can't use docker my phone isn't rooted
